backend/Squirrel.Core/Squirrel.Core.BLL/Services/ProjectService.cs

```diff
@@ -7,70 +7,81 @@
 using Squirrel.Core.DAL.Entities;
 using Squirrel.Shared.Exceptions;
 
-namespace Squirrel.Core.BLL.Services;
-
-public sealed class ProjectService : BaseService, IProjectService
-{  
-    private readonly IUserIdGetter _userIdGetter;
-    public ProjectService(SquirrelCoreContext context, IMapper mapper, IUserIdGetter userIdGetter) : base(context, mapper)
+namespace Squirrel.Core.BLL.Services
+{
+    public class ProjectService : BaseService, IProjectService
     {
-        _userIdGetter = userIdGetter;
-    }
+        private readonly IUserIdGetter _userIdGetter;
 
-    public async Task<ProjectDto> AddProjectAsync(ProjectDto projectDto)
-    {
-        var project = _mapper.Map<Project>(projectDto)!;
-        var createdProject = (await _context.Projects.AddAsync(project)).Entity;
-        
-        await _context.SaveChangesAsync();
-        
-        return _mapper.Map<ProjectDto>(createdProject)!;
-    }
+        public ProjectService(SquirrelCoreContext context, IMapper mapper, IUserIdGetter userIdGetter) : base(context,
+            mapper)
+        {
+            _userIdGetter = userIdGetter;
+        }
 
-    public async Task<ProjectDto> UpdateProjectAsync(int projectId, ProjectDto projectDto)
-    {
-        var existingProject = await _context.Projects.FindAsync(projectId);
-        if (existingProject is null)
+        public async Task<ProjectDto> AddProjectAsync(ProjectDto projectDto)
         {
-            throw new EntityNotFoundException();
+            var projectEntity = _mapper.Map<Project>(projectDto);
+
+            await _context.Projects.AddAsync(projectEntity);
+
+            await _context.SaveChangesAsync();
+
+            return _mapper.Map<ProjectDto>(projectEntity);
```

Comment:
Won't it be possible to create project for different user? And the same with updating